### ISSUE TITLE
undefined symbol: _ZN9Microsoft3MSR4CNTK8MBLayout13s_nameIndicesB5cxx11E

### ISSUE THREAD

**Issue by Prasandhmcw** [Microsoft/CNTK#2102]
Hi, 

I am trying to run an EndToEnd Test but ending up with the following : 

**Command :**

python TestDriver.py run BatchNormalization/NonSpatial/CNTK --flavor=release

**Error log :**

symbol lookup error: /home/user/cntk/CNTK/build/release/bin/cntk: undefined symbol: _ZN9Microsoft3MSR4CNTK8MBLayout13s_nameIndicesB5cxx11E

My system is Ubuntu 16.04 with GeForce GTX 860M , Cuda 8.0 

Any leads ?

**Comment by cha-zhang** [comment]
Could you try: 
python TestDriver.py run BatchNormalization/NonSpatial/CNTK -d gpu -s gpu -f release

**Comment by Prasandhmcw** [comment]
Hey cha-zhang ,

Thanks for the input. It has fixed the issue. Does this mean that only gpu tests are available for these ?

**Comment by cha-zhang** [comment]
Yes, batch normalization is GPU only for now.
